tests/python/unittest/test_loss.py

```diff
@@ -70,15 +70,14 @@ def test_ce_loss():
     label = mx.nd.array(np.random.randint(0, nclass, size=(N,)), dtype='int32')
```

Comment:
Submit test fix in a separate PR and revert the change on file property
100644 → 100755

Reply:
resolved